Documentation: minor additional doc fixes

jrfnl opened the pull request:
Fixes a few parameter comments.

moorscode commented on src/messages/Whip_InvalidVersionRequirementMessage.php:
```diff
@@ -16,15 +16,15 @@ class Whip_InvalidVersionRequirementMessage implements Whip_Message {
 
 	/**
-	 * @var
+	 * @var int
```
As this parameter is holding a `version` number (4.1.2), the type of the variable should be `string`

moorscode commented on src/messages/Whip_InvalidVersionRequirementMessage.php:
```diff
@@ -16,15 +16,15 @@ class Whip_InvalidVersionRequirementMessage implements Whip_Message {
 	 *
 	 * @param Whip_VersionRequirement $requirement
-	 * @param                         $detected
+	 * @param int                     $detected
```
As this parameter is holding a `version` number (4.1.2), the type of the variable should be `string`

jrfnl commented:
@moorscode Did you see my note about this in the commit message ?

> Based on the fact that this class is only called from the `Whip_RequirementsChecker::addMissingRequirementMessage()` method and in that function call the return value of a call to the `Whip_Configuration::configuredVersion()` method is passed and that the `Whip_Configuration::configuredVersion()` method according to its documentation returns an integer.

If it's `string`, existing documentation is wrong and needs to be fixed.

jrfnl commented:
@moorscode OK, so I've dug a little deeper and agree with you that it can *also* be a string, so I have updated the two new parameter/property type comments, added a short description to both and I have adjusted the existing documentation on which I'd based the original values.

I've done this in a separate commit for easier review. Please let me know if you want me to squash this last commit into the previous one.
